code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -53,6 +53,32 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	tool_qualities = list(QUALITY_CUTTING = 20)
 
+/obj/item/tool/makeshiftaxe
+	name = "makeshift axe"
+	desc = "A heavy plasteel blade affixed to a welded metal shaft, for close up carnage"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "makeshift_axe"
+	item_state = "makeshift_axe"
+	wielded_icon = "makeshift_axe_wielded"
+	matter = list(MATERIAL_STEEL = 2, MATERIAL_PLASTEEL = 1)
+	worksound = WORKSOUND_HARD_SLASH
+	force = WEAPON_FORCE_DANGEROUS
+	throwforce = WEAPON_FORCE_NORMAL
+	armor_divisor = ARMOR_PEN_MODERATE
+	w_class = ITEM_SIZE_NORMAL
+	slot_flags = SLOT_BACK
+	sharp = TRUE
+	edge = TRUE
+	attack_verb = list("chopped", "torn", "cut", "cleaved", "slashed")
+	hitsound = 'sound/weapons/melee/heavystab.ogg'
+	tool_qualities = list(QUALITY_CUTTING = 20)
```

Comment:
```suggestion
	tool_qualities = list(QUALITY_CUTTING = 10)
```
Cutting 20 is kitchen knife level. This is a makeshift axe, not made for cutting bread.

Reply:
That's fair, didnt consider that